tests/dummy/config/environment.js

```diff
@@ -7,6 +7,7 @@ module.exports = function(environment) {
     environment,
     rootURL: '/',
     locationType: 'auto',
+    originForOverride: 'http://localhost:3000',
```

Comment:
Any reason to make this configurable? I know it's used in both the route and the test, but I think it might be better to hardcode in those places. I'd like to avoid a situation where someone thinks this is a generic config var since it's really only used for testing this feature.

Reply:
This was simply to be sure the correct host was passed through to the test. Happy to configure however makes sense.